snomed/com.b2international.snowowl.snomed.reasoner/plugin.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
```

Comment:
Would it be possible to use the newer plugin service registration mechanism? ClassPathScanner + service/toolingId lookup.

Reply:
I've replaced the extension point with `ClassPathScanner` and a registry in commit cfef3f5.